includes/Core/Authentication/Authentication.php

```diff
@@ -182,6 +182,15 @@ final class Authentication {
 	 */
 	protected $google_proxy;
 
+	/**
+	 * Initial_Version instance.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @var User_Input_State
+	 */
+	private $initial_version = null;
```

Comment:
The `@var` definition needs updating here. Also, the IB (as well as other instance properties here) are `protected` but this is `private`. Also, let's omit the `null` assignment for these since that is the value it will have simply by declaring it anyways.
```suggestion
	 * @var Initial_Version
	 */
	protected $initial_version;
```
The empty line between `@since` and `@var` is removed in the suggestion as well for consistency with the others.